### ISSUE TITLE
Screen grabbing work only background problem

### ISSUE THREAD

**Issue by akvarist76** [psieg/Lightpack#534]
On mac mini (m1), screen grabbing only works in desktop background. It works when I change the background photo, but not when I open a video, photo or app from anywhere.  Screen recording access enabled.

Mac os monterey
Prismatik 5.11.2.31

**Comment by Stratos125** [comment]
Hi,

I have the same issue, MacBook Pro M1 16" running Montery 12.4.  I've tried the universal 5.11.2.27 and 5.11.2.31 versions with the same results.  Only grabs the background.

**Comment by zomfg** [comment]
can you try [this one](https://github.com/zomfg/Lightpack/releases/download/5.11.2.24/Prismatik_5.11.2.31_universal.dmg)

**Comment by Stratos125** [comment]
Hey, thanks for the link, but now it's a slightly different issue.  Screen Grabbing doesn't update the LED's at all.  I am running Montery 12.5 now too, have screen recording permissions enabled, and I have tried resigning the app's certificate as well.  Mood lamp works, so the software is able to drive the LED's fine, just no grabby grabby. :(

Not sure if it helps at all, but when switching from mood lamp back to screen grab, the LED's hold the last colours from mood lamp for about 60 seconds before switching them off.

**Comment by zomfg** [comment]
does it say 0 fps?
try closing prismatik, removing screen recording permission, start prismatik, allow screen recording but click "later" when asked to restart, then quit/restart prismatik manually

**Comment by Stratos125** [comment]
Yeah said 0.  Tried your steps above and it's working a treat.  Thank you so much for continuing to support this! :)

**Comment by zomfg** [comment]
does it work if you do this with the normal build?

**Comment by Stratos125** [comment]
I got it working on the normal build.  Had to take a few steps though so what I did was:
1. Remove Prismatik from Screen Recording permissions
2. Uninstall existing version
3. Install the normal 5.11.2.31 build for Mac
4. Resign the Prismatik certificate with one with one in my Apple keychain
5. Acknowledge the unverified developer bits with Ctrl + Open
6. Start Prismatik
7. Allow screen recording but click "later" when asked to restart
8. Quit Prismatik manually
9. Reboot my Mac
10. Presto, it's working on the normal build!!! 🥇

**Comment by tarios** [comment]
Hi
i tried the procedure above but it doesn't work, still it shows ffps 0, when macAV is checked, when mac CG is checked, ffps is >0 but leds doesn't working, i have mac mini with macOS 12.6
I think that it is not well installed either, when I turn off the computer, the program disappears from the menu bar, I have to install it again,
how can I Prismatik certificate be changed?
